test/infoSpec.js

```diff
@@ -33,7 +33,7 @@ describe('Info', () => {
     this.mockSocket
       .reply(btpPacket.TYPE_MESSAGE, ({ requestId }) => btpPacket.serializeResponse(requestId, []))
 
-    await this.plugin.addSocket(this.mockSocket, { username: 'user', token: 'placeholder' })
+    await this.plugin.addSocket(this.mockSocket, { username: '', token: 'placeholder' })
```

Comment:
`username` and `token` are the same as what is defined in options.server. Better to declare a constant for username and token at the top and reference the constant.

Thinking about it some more, is there any reason `username` and `token` have to be passed to `addSocket`? The plugin could parse this information from the `options.server`, no?